.circleci/images/node-6/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM node:6
+RUN apt-get install fonts-dejavu-core
```

Comment:
You need to update the package list too, otherwise it might install an old version. It's also good to clear the cache, to avoid bloating out the size of the Docker file with unnecessary caches. I normally do something like this:
```
RUN apt-get -y update && \
  apt-get install -y --no-install-recommends fonts-dejavu-core && \
  apt-get clean && \
  rm -rf /var/lib/apt/lists/*
```

Reply:
Thanks!